scripts/setup_admin_user

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
How about starting `db` service from the script itself?

Reply:
Because it's not necessarily for the `db` service, in fact I did this to set up a new database in Atlas.

Reply:
Oh, yes. Mongo URL can be anything and not just the local `db` service.